docs/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -4,64 +4,106 @@ title: "Red Hat OpenShift"
 
-| OpenShift version | Supported          |
-| ----------------- | ------------------ |
-| 4.12.x            | :white_check_mark: |
-| 4.13.x            | :white_check_mark: |
-| 4.14.x            | :white_check_mark: |
+## Cluster Specification
 
-Any version not explicitly marked in the table above is not tested, and we cannot guarantee compatibility.
+When deploying Camunda 8 on an OpenShift cluster, the cluster specification should align with your specific requirements and workload characteristics. Here's a suggested configuration to begin with:
 
-## Pitfalls to avoid
+- **Instance type:** 4 vCPUs (x86_64, >3.1 GHz), 16 GiB Memory (for example, [m5.xlarge on AWS](https://aws.amazon.com/en/ebs/general-purpose/))
+- **Number of dedicated nodes:** 4
+- **Volume type:** SSD volumes (with between 1000 and 3000 IOPS per volume, and a throughput of 1,000 MB/s per volume, for instance, [gp3 on AWS](https://aws.amazon.com/en/ebs/general-purpose/))
 
-For general deployment pitfalls, visit the [deployment troubleshooting guide](/self-managed/operational-guides/troubleshooting/troubleshooting.md).
+### Supported Versions
 
-### Security Context Constraints (SCCs)
+We conduct testing and ensure compatibility against the following OpenShift versions:
 
-Much like how roles control the permissions of users, Security Context Constraints (SCCs) are a way to control the permissions of the applications deployed, both at the pod and container level. It's generally recommended deploying your application with the most restricted SCCs possible. If you're not familiar with security context constraints, refer to the [OpenShift documentation](https://docs.openshift.com/container-platform/latest/authentication/managing-security-context-constraints.html).
+- Any version not listed below hasn't been tested, and we do not guarantee compatibility.
+- The OpenShift end of support date must be in the future: we do not ensure compatibility with versions no longer supported by Red Hat, as indicated in the "End of Support Date" column.
 
-#### Permissive SCCs
+| OpenShift Version | [End of Support Date](https://access.redhat.com/support/policy/updates/openshift) |
+| ----------------- | --------------------------------------------------------------------------------- |
+| 4.15.x            | August 27, 2025                                                                   |
+| 4.14.x            | May 1, 2025                                                                       |
+| 4.13.x            | November 17, 2024                                                                 |
+| 4.12.x            | July 17, 2024                                                                     |
 
```

Comment:
I'm wondering how this may clash with our [Maintenance Gurantuee](https://confluence.camunda.com/display/HAN/Camunda+8+Maintenance+Guarantees) of 18 months?

E.g. 8.5 was released but it would by EOL not be supported anymore by any OpenShift version that was present when it was released.

Reply:
but I guess by saying `- The OpenShift end of support date must be in the future: we do not ensure compatibility with versions no longer supported by Red Hat, as indicated in the "End of Support Date" column.` we cover that case and just say, dependency faults? 🤷